network/src/commonTest/kotlin/com/wire/kalium/api/v4/UserDetailsApiV4Test.kt

```diff
@@ -68,6 +68,8 @@ internal class UserDetailsApiV4Test : ApiTest() {
         val params = ListUserRequest.qualifiedIds(
             listOf(QualifiedIDSamples.one, QualifiedIDSamples.two, QualifiedIDSamples.three)
         )
+
+        println(SUCCESS_RESPONSE.rawJson)
```

Comment:
I guess this is a left over for debugging purposes

Reply:
Deleted.